Python/initconfig.c

```diff
@@ -2143,10 +1974,7 @@ config_init_import(PyConfig *config, int compute_path_config)
     if (config->use_frozen_modules < 0) {
         const wchar_t *value = config_get_xoption_value(config, L"frozen_modules");
         if (value == NULL) {
-            int isdev = is_dev_env(config);
-            if (isdev >= 0) {
-                config->use_frozen_modules = !isdev;
-            }
+            config->use_frozen_modules = !config->_development_env;
```

Comment:
I purposefully left it -1 if we couldn't figure it out.  So this change implies that we are always able to figure it out.  Is that the case?

Reply:
> we'll assume it's a dev build.

That's what I did at first too.  However, I found that it was helpful to know if we were not able to figure it out, which is what the -1 indicates in this case.

Reply:
I mean, we either find the build landmarks or we don't... what other checks do you have? If the build landmarks aren't there, it's not a dev build, and if they are but the stdlib isn't, it's all going to crash down anyway.